cirq-core/cirq/value/classical_data.py

```diff
@@ -133,27 +143,27 @@ def __init__(
             _measured_qubits = {}
         if _channel_measurements is None:
             _channel_measurements = {}
-        self._measurements: Dict['cirq.MeasurementKey', Tuple[int, ...]] = _measurements
+        self._measurements: Dict['cirq.MeasurementKey', List[Tuple[int, ...]]] = _measurements
         self._measured_qubits: Dict[
-            'cirq.MeasurementKey', Tuple['cirq.Qid', ...]
+            'cirq.MeasurementKey', List[Tuple['cirq.Qid', ...]]
         ] = _measured_qubits
-        self._channel_measurements: Dict['cirq.MeasurementKey', int] = _channel_measurements
+        self._channel_measurements: Dict['cirq.MeasurementKey', List[int]] = _channel_measurements
         self._measurement_types: Dict[
             'cirq.MeasurementKey', 'cirq.MeasurementType'
         ] = _measurement_types
 
     @property
-    def measurements(self) -> Mapping['cirq.MeasurementKey', Tuple[int, ...]]:
+    def measurements(self) -> Mapping['cirq.MeasurementKey', List[Tuple[int, ...]]]:
```

Comment:
@95-martin-orion you were wanting to change this to `records`, correct? Did you have any preferences for the other properties below?

Reply:
Consistency with #4555 is my only real concern here - so `channel_records` would be appropriate. Everything else seems fine to me.